src/referral/referral_view.py

```diff
@@ -0,0 +1,69 @@
+from datetime import datetime, timedelta
+
+from rest_framework import status, viewsets
+from rest_framework.decorators import action
+from rest_framework.permissions import IsAuthenticated
+from rest_framework.response import Response
+
+from referral.related_models.referral_invite import ReferralInvite
+from referral.serializers.referral_invite_serializer import ReferralInviteSerializer
+from researchhub.settings import TESTING
+from user.related_models.user_model import User
+from utils.http import POST
+
+from .related_models.referral_invite import BOUNTY, JOIN_RH
+
+
+class ReferralViewSet(viewsets.ModelViewSet):
+    @action(detail=False, methods=[POST], permission_classes=[IsAuthenticated])
+    def invite(self, request):
```

Comment:
Let's standardize this into a more traditional model view.
The recent `bounty_view.py` is a good example. It includes the permission classes, queryset, serializer, etc.

I think the `invite` endpoint is unnecessary, as the default `create` is essentially the same method.

Reply:
I changed from `ReferralViewSet` => `ReferralInviteViewSet` and now using create + serializer.
LMK if there is anything in particular you want me to pull from the bounty_view.